ifrs17/Import/ImportScopeCalculation.ipynb

```diff
@@ -2316,7 +2323,55 @@
         {
             "cell_type": "code",
             "source": [
-                "public interface ComputeIfrsVarsOpenings : ActualToIfrsVariable, DeferrableToIfrsVariable, TmToIfrsVariable",
+                "public interface ProjectionScopeCashflows : IScope<ImportIdentity, ImportStorage>{",
```

Comment:
imo there's no need to rename this scope

Reply:
There exist another scope with the same name:

![image](https://user-images.githubusercontent.com/120557622/221876595-1081369f-889a-4a0c-9384-1e7c8e7f2106.png)

Reply:
@dcolleoni  Done.